product/dialogs/miq_dialogs/miq_provision_vmware_dialogs_template.yaml

```diff
@@ -435,7 +435,7 @@
           :auto_select_single: false
           :description: Name
           :required_method: :validate_placement
-          :required: true
+          :required: false
```

Comment:
Since this is no longer required, the `:required_method:` will not run.  I would suggest changing it to a `validation_method` and verify that `validate_placement` returns the expected result with and without a host selected.